k2/csrc/context_inl.cuh

```diff
@@ -0,0 +1,182 @@
+/**
+ * @brief
+ * context_inl
+ * this file is supposed to be included only by context.h
+ * It' the implementation of template functions and inlined functions.
+ *
+ * @copyright
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey
+ *                                                   Meixu Song)
```

Comment:
seeems you forgot the orignal author list (OK, me.. as I implement those memory copy and context CHECK template) by moving code to this file..

Reply:
sorry, get you added here.
But I checked the git history, it may not be my mistake. It was all the time?

Reply:
just after I add those `MemoryCopy` functions